src/pages/lib/in-app-messaging/clear-messages/q/platform/[platform].mdx

```diff
@@ -0,0 +1,8 @@
+export const meta = {
+  title: `Clear messages`,
+  description: `Amplify In-App Messaging helps you better engage your users with messages they can see while actively using your application. Send targeted messages to your defined user segments or even trigger contextual messages based on user behavior. Once you are up and running with In-App Messaging, you’ll be able to create messages that look native to your application and deliver them to your users all without additional code changes.`
```

Comment:
Not sure if there is a pre-defined pattern for constant usage in the docs, but we should be assigning this to a constant instead of replicating it (10 more times) times in this PR. For example, if we needed to update the value here, the replicas could be missed

Reply:
We probably need a  unique meta description for each page for better seo. I'll add the meta descriptions on monday.